Add depolyment to Travis builds

mithro opened the pull request:
* Deploy markdown pages to Github pages (including generated output from verilog_lint). See https://mithro.github.io/verible/ for example output.
 * Deploy precompiled binaries to Github releases. See https://github.com/mithro/verible/releases for example output.

hzeller commented:
This looks very neat, also thanks for creating the lint documentation.
Awaiting green-ness of builds ...

hzeller commented on .travis.yml:
```diff
@@ -1,22 +1,90 @@
+  - bazel --version
 
 script:
   - bazel build --noshow_progress --cxxopt='-std=c++17' //...
```
The c++17 should now (after ca3fce403ab5bf2cf56795b747da718c2be2b3ca )  work with C++11

mithro commented:
@hzeller Please wait until I prove the deployment actually works.... Still testing at https://travis-ci.org/mithro/verible

hzeller commented:
https://travis-ci.org/mithro/verible looks good now, ready to merge ?

fangism commented on .travis.yml:
is this email address valid? or what we intend?

mithro replied:
I changed it to the developer mailing lists.

mithro commented on .travis.yml:
Done.

mithro commented on .travis.yml:
Done.

hzeller commented on AUTHORS:
```diff
@@ -3,7 +3,12 @@
 # some cases, their employer may be the copyright holder.  To see the full list
 # of contributors, see the revision history in source control.
+
+# Companies
```
If this was a separate pull request, it should be more visible.

mithro replied:
Pushed into a separate pull request.

mithro commented:
@hzeller This change should now be ready to merge!

 - [x] Output at https://mithro.github.io/verible looks pretty good.
 - [x] Binaries are being uploaded to https://github.com/mithro/verible
 - [x] Travis isn't building the automatically generated tags.

hzeller commented:
Can you add copyright headers to all new files ? I think github-pages-setup.sh, github-releases, install-bazel, output-log and set-compiler are affected.

It should contain at least somewhere `Copyright 2020 The Verible Authors` in it.

Also github-pages-setup.sh and github-releases-setup.sh contains tabs, but shall contain spaces.

mithro commented:
> Can you add copyright headers to all new files ? I think github-pages-setup.sh, github-releases, install-bazel, output-log and set-compiler are affected.
>
> It should contain at least somewhere `Copyright 2020 The Verible Authors` in it.

Fixed.
 
> Also github-pages-setup.sh and github-releases-setup.sh contains tabs, but shall contain spaces.

Fixed.

hzeller commented:
mmh, looks like there are also some files in the kokoro/ subdirectory that also need copyright headers.

mithro commented:
> mmh, looks like there are also some files in the kokoro/ subdirectory that also need copyright headers.

I've done those too, right?

mithro commented:
I pushed an updated copyright line to include a full stop as the presubmit seems to require?

hzeller commented:
ah, yes, looks good.

hzeller commented:
Is the recent failure in the conda-package build because of changes in the deployment outputs ?

It complains about 
`Error: bad character '-' in package/version: v0.0-190-g07d75ca_0000_g07d75ca
2483`

https://travis-ci.com/SymbiFlow/conda-packages/jobs/284095212
